README.md

```diff
@@ -157,6 +157,11 @@ ratelimit:
   #  - 123.456.789.233
   #  - 456.789.90.2
 
+# DNS caching settings
+dns_cache:
+  ttl: 5 # Time-to-live for cache entries, in seconds
+  size: 10 # Optional cache size, default: 100
```

Comment:
I see. Since examples are often copy / pasted around, lets perhaps keep it at 100, to avoid surprises with bad performance?